osu.Game.Rulesets.Osu/UI/OsuResumeOverlay.cs

```diff
@@ -38,7 +38,19 @@ public override void Show()
             clickToResumeCursor.ShowAt(GameplayCursor.ActiveCursor.Position);
 
             if (localCursorContainer == null)
-                Add(localCursorContainer = new OsuCursorContainer());
+            {
+                var newContainer = new OsuCursorContainer();
+                Add(localCursorContainer = newContainer);
+
+                clickToResumeCursor.CursorScale = newContainer.CalculatedCursorScale.Value;
+                clickToResumeCursor.Scale = new Vector2(newContainer.CalculatedCursorScale.Value);
+
+                newContainer.CalculatedCursorScale.ValueChanged += e =>
+                {
+                    clickToResumeCursor.CursorScale = e.NewValue;
+                    clickToResumeCursor.Scale = new Vector2(e.NewValue);
+                };
```

Comment:
Possibly worth factoring the two-liners out into a method and calling that instead?

Reply:
I don't think this should be factored out, it's only 2 lines that are only duplicated once. Even making it a local method would be more complicated than needed.

Reply:
You can use `BindValueChanged(delegate, true)` to solve this. it's used in hundreds of places across the codebase.